spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/AbstractMessageListenerContainer.java

```diff
@@ -322,6 +353,23 @@ public void setDeBatchingEnabled(boolean deBatchingEnabled) {
 	}
 
 	/**
+	 * Public setter for the {@link Advice} to apply to listener executions. If {@link #setTxSize(int) txSize>1} then
```

Comment:
Looks like we don't have `setTxSize()` here. That is an option for the simple container.

(And yes, GH has come back to life :smile: )